src/amo/pages/UserProfile/index.js

```diff
@@ -416,6 +419,18 @@ export class UserProfileBase extends React.Component<InternalProps> {
                 {i18n.gettext('Edit profile')}
               </Button>
             ) : null}
+
+            {canAdminUser && user ? (
+              <Button
+                className="UserProfile-admin-link"
+                buttonType="neutral"
+                href={`/admin/models/users/userprofile/${user.id}/`}
```

Comment:
I know in the issue it says this url won't work, right? It wasn't clear to me why tho...
Anyway, I'm just wondering if this is something we should hide till it does work?

Reply:
I think @wagnerand was just pointing out that the `username` (he said "slug") won't work and that's why we needed to use the `id` instead. The link does work - I tested it.

Reply:
hmm doesn't work for me. Is it behind the vpn? maybe that's why I can't see it...

Reply:
Yes, it's behind the VPN